tests/test_compile.py

```diff
@@ -1,9 +1,9 @@
-from . import c_eval
-from . import c_exec
-from . import e_eval
 from RestrictedPython import compile_restricted
 from RestrictedPython import CompileResult
 from RestrictedPython._compat import IS_PY2
+from tests import c_eval
+from tests import c_exec
+from tests import e_eval
```

Comment:
Could you please explain why you moved from relative import to absolute import ?

To be honest, I don't like code in `__init__.py` at all.
What about moving all the code from `__init__.py` into a `helpers.py` and then use a relative import
 `from .helpers import c_eval`.
I guess there are better names than `helpers.py', I used that one to express the idea.
What do you think ?